lib/spack/spack/relocate.py

```diff
@@ -379,6 +459,8 @@ def replace_prefix_text(path_name, old_dir, new_dir):
     in text files using utf-8 encoded strings.
     """
     with open(path_name, 'rb+') as f:
+        tty.debug('In replace_prefix_text(%s,%s,%s)' % (path_name, old_dir,
```

Comment:
This seems slightly low-level even for a debug statement: does this help with something in particular? No problem if it lives here until the end of the PR, but it may not be useful after that.

Reply:
I will remove them. It makes the -d output very verbose.